Entities/Structures/Door/SwingDoor.as

```diff
@@ -149,6 +149,10 @@ void onCollision(CBlob@ this, CBlob@ blob, bool solid)
 	if (blob !is null)
 	{
 		this.getCurrentScript().tickFrequency = 3;
+		if (!isOpen(this) && canOpenDoor(this, blob)) 
```

Comment:
Why is this needed here and what consequences does it have vs the current behavior?  
Is it just a cleanup of what is otherwise done in `doesCollideWithBlob`?

Reply:
yes, I removed that part from doesCollideWithBlob and put it here - more logical this way imo